### PR TITLE
Add console access

### PR DESCRIPTION
**Pull request by cviecco**

### CODE REVIEW

File: broker/api.go [hunk 1]
```diff
@@ -13,4 +13,6 @@ type PermittedAccount struct {
 type Broker interface {
 	UpdateConfiguration(config *configuration.Configuration) error
 	GetUserAllowedAccounts(username string) ([]PermittedAccount, error)
+	UserAllowedToAssumeRole(username string, accountName string, roleName string) (bool, error)
```
[the remark this answers is not in the document]
Comment (zoresvit) [reply broker/api.go]: It adds verbosity, but not clarity :) 

A common pattern in many languages for naming a function returning Boolean is by prefixing it with `is`:
`IsUserAllowedToAssumeRole`.

[the remark this answers is not in the document]
Comment (rgooch) [reply broker/api.go]: I'm OK with the `Is` prefix, too.

[the remark this answers is not in the document]
Comment (cviecco) [reply broker/api.go]: I'll take ruslan's suggestion.

File: broker/aws/impl.go [hunk 2]
```diff
@@ -119,7 +121,7 @@ func (b *Broker) withSessionGetAWSRoleList(validSession *session.Session) ([]str
 }
 
 func (b *Broker) masterGetAWSRolesForAccount(accountName string) ([]string, error) {
-	assumeRoleOutput, err := b.withProfileAssumeRole(accountName, masterAWSProfileName, masterRoleName)
+	assumeRoleOutput, err := b.withProfileAssumeRole(accountName, masterAWSProfileName, masterRoleName, "brokermaster")
```
[the remark this answers is not in the document]
Comment (cviecco) [reply broker/aws/impl.go]: The function withProfileAssumeRole (lines 82 &83) was made to be more generic.  Getting the list with the master role is still needed, but only within this function. This is a marker for aws logs to tag the cloudtrail log. I can be any value, but "brokermaster" is the value of the old broker.

File: broker/httpd/api.go [hunk 3]
```diff
@@ -80,6 +80,7 @@ func StartServer(staticConfig *staticconfiguration.StaticConfiguration,
 	http.HandleFunc("/status", server.statusHandler)
 	serviceMux := http.NewServeMux()
 	serviceMux.HandleFunc("/", server.consoleAccessHandler)
+	serviceMux.HandleFunc("/getconsole", server.getConsoleUrlHandler)
```
[the remark this answers is not in the document]
Comment (cviecco) [reply broker/httpd/api.go]: Will not rename now, this will break the client. After the client code is landed we can rename in both

**Pull request merged**